src/annotator/integrations/html-side-by-side.js

```diff
@@ -108,6 +108,21 @@ function hasFixedPosition(element) {
   }
 }
 
+/**
+ * Return the bounding rect that contains the element's content, including
+ * any content which overflows the element's specified size.
```

Comment:
Is it possible to expand this comment in some way that draws a connection between "overflows the element's specified size" and "exceeds dimensions of the element's BoundingClientRect"? That wasn't immediately obvious when I looked at this at first. This could also be tackled by comments in the function itself.